corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -24,6 +24,15 @@ def get_all_usernames_by_domain(domain):
     return (row['key'][3] for row in _get_all_user_rows(domain, include_web_users=True))
 
 
+def get_all_user_id_username_pairs_by_domain(domain, include_web_users=True, include_mobile_users=True):
```

Comment:
nit: I think if you return a list of the `UserInfo` namedtuple's this might be a bit easier to reason about and use, but this seems fine too.